packages/mobile/src/send/SendConfirmation.tsx

```diff
@@ -176,34 +181,34 @@ class SendConfirmation extends React.Component<Props, State> {
     this.sendOrInvite(InviteBy.SMS)
   }
 
-  render() {
+  renderWithAsyncFee: CalculateFeeChildren = (asyncFee) => {
     const { t, appConnected, isSending } = this.props
-    const { amount, reason, recipient, recipientAddress, fee } = this.getConfirmationInput()
+    const { amount, reason, recipient, recipientAddress } = this.getConfirmationInput()
 
     const currentBalance = this.props.dollarBalance
-    const amountWithFees = new BigNumber(numeral(amount).value()).plus(
-      this.props.suggestedFeeDollars
-    )
-    const userHasEnough = amountWithFees.isLessThanOrEqualTo(currentBalance)
+    const fee = asyncFee.result && getFeeDollars(asyncFee.result)
+    const amountWithFee = new BigNumber(numeral(amount).value()).plus(fee || 0)
```

Comment:
just to be clear - the intention here would be to show the amount the user has already entered while we are calculating the fee, and then update once the fee has been calculated? just wondering if seeing a number jump would be a bad UX, and we should just show loading instead (which is maybe covered by that isLoadingFee?)

Reply:
Good point. 
What's happening right now is the send fee is so small it displays as `$0.0000` after loading and the total doesn't change.
The invite fee though is more but since it's [currently hardcoded](https://github.com/celo-org/celo-monorepo/blob/f272bc826af3bf73843f9d06cfe2da181e63cc1c/packages/mobile/src/invite/saga.ts#L57-L60), there's no loading indicator happening.

With the way the UI is done currently, I think that fact that the total can change after the fee has loaded shouldn't be surprising for the user.
But we need to confirm this. If it's a problem we can tweak this.